assets/charts/components/linkerd2/templates/service-monitors.yaml

```diff
@@ -1,3 +1,16 @@
+# Following relabel config is taking from:  https://linkerd.io/2/tasks/external-prometheus/
```

Comment:
Should this be contributed upstream?

Reply:
I am not sure if they are willing to take "prometheus-operator" related config. https://github.com/linkerd/website/issues/853